lib/crystalball/rspec/runner/configuration.rb

```diff
@@ -7,9 +7,10 @@ module RSpec
     class Runner
       # Class for storing local runner configuration
       class Configuration
-        def initialize(config = {})
+        def initialize(config = {}) # rubocop:disable Metrics/MethodLength
           @values = {
-            'map_path' => 'tmp/execution_maps',
+            'execution_map_path' => 'tmp/execution_map.yml',
+            'tables_map_path' => 'tmp/tables_map.yml',
```

Comment:
I don't think we need default values here since this tables_map file is optional to one strategy only